test/src/deploy.test.js

```diff
@@ -0,0 +1,24 @@
+require('../../src/setup');
+const deploy = require('../../src/deploy');
+const shouldFail = require('../../src/shouldFail');
+const { ERC1820_REGISTRY_BYTECODE } = require('../../src/data');
+
+contract('deploy', function ([deployer]) {
```

Comment:
`deploy` isn't really a contract. Wouldn't it be ok to just do the following?
```suggestion
describe('deploy', function ([deployer]) {
```

Reply:
No, because we need to get truffle to initialize a ganache instance and send us the accounts array, and all of that is triggered by `contract`.